terracotta/handlers/metadata.py

```diff
@@ -47,7 +47,7 @@ def multiple_metadata(
 
     out = []
     with driver.connect():
-        for dataset in datasets:
+        for dataset in datasets[:settings.MAX_POST_METADATA_KEYS]:
```

Comment:
Can we throw a 400 error instead? Silently returning less data than requested will net us some angry issues :)

Reply:
Good catch 😄